src/py/aspen/database/models/sequences.py

```diff
@@ -84,14 +90,14 @@ class SequencingProtocolType(enum.Enum):
 )
 
 
-class SequencingReadCollection(Entity):
-    __tablename__ = "sequencing_read_collections"
+class SequencingReadsCollection(Entity):
+    __tablename__ = "sequencing_reads_collections"
     __table_args__ = (UniqueConstraint("s3_bucket", "s3_key"),)
     __mapper_args__ = {"polymorphic_identity": EntityType.SEQUENCING_READS}
 
     entity_id = Column(Integer, ForeignKey(Entity.id), primary_key=True)
     sample_id = Column(Integer, ForeignKey(Sample.id), nullable=False)
-    sample = relationship(Sample, backref=backref("sequencing_reads"))
+    sample = relationship(Sample, backref=backref("sequencing_reads_collection"))
```

Comment:
should this be `sequencing_reads_collections`?

Reply:
Miraculously, this is correct.  For any given sample, there can be at most one `sequencing_reads_collection`.

Reply:
ok interesting lol